### DISCUSSION TITLE
Workshop - Dates and Times

### DISCUSSION THREAD

**Discussion by paulboyes** [COVESA/data-expert-group#5] [Polls]
As discussed today in our weekly Data Expert Group meeting, hourly meetings weekly are good but their is not enough time to deep dive into topics.  It was agreed that quarterly workshops (mainly remote, but could be in person or following an AMM)  that are a minimum 4 hours long, for 1 or 2 days, would give us the time to organize and deep dive.  We discussed having the first workshop in early February or even late January.  

What dates/times do we want to have the first workshop?  The current thinking is the first one should be 2 days on Wednesday and Thursday for 4 hours a day.  Covering four time zones is a challenge.  I know the below time is not ideal for everyone, but should work. 

5 - 9am PT
1400-1700 CET
6:30-10:30pm IST (Thursday and Friday)
10pm - 2am JST

Please choose what works best for you.  If you have other thoughts or suggestions as to what will work best, please let the group know.

**Comment by paulboyes** [comment]
Sooner is better I think.

**Reply by paulboyes** [reply]
@adobekan @erikbosch @slawr @tguild @tspreck @SebastianSchildt @UlfBj @jobyjester please take the survey and comment if you have comments.

**Comment by erikbosch** [comment]
All of them works for me

**Reply by tguild** [reply]
same

**Comment by UlfBj** [comment]
Feb 8-9 looks complicated for me.

**Comment by erikbosch** [comment]
@paulboyes  - the text above mentioning "6:30-10:30pm IST (Thursday and Friday)" is not correct, right? It should be Wednesday and Thursday also in India?
